src/script/auth/page/Login.tsx

```diff
@@ -120,11 +121,18 @@ const LoginComponent = ({
 
   const isOauth = UrlUtil.hasURLParameter(QUERY_KEY.SCOPE, window.location.hash);
 
+  const features = Config.getConfig().FEATURE;
+
   const [showEntropyForm, setShowEntropyForm] = useState(false);
-  const isEntropyRequired = Config.getConfig().FEATURE.ENABLE_EXTRA_CLIENT_ENTROPY;
+  const isEntropyRequired = features.ENABLE_EXTRA_CLIENT_ENTROPY;
   const onEntropyGenerated = useRef<((entropy: Uint8Array) => void) | undefined>();
   const entropy = useRef<Uint8Array | undefined>();
 
+  const queryConversationCode = UrlUtil.getURLParameter(QUERY_KEY.CONVERSATION_CODE) || null;
+  const queryConversationKey = UrlUtil.getURLParameter(QUERY_KEY.CONVERSATION_KEY) || null;
+
+  const keyAndCodeExistent = queryConversationKey && queryConversationCode;
```

Comment:
rather that adding some implicit dependencies here, let's rather use the `embedded` flag set to the `Login` form to decide if we should do the redirect or not. This should be cleaner (and the ConversationJoin page does set this flag)

Reply:
nice, sounds much better